tests/callbacks/test_finetuning_callback.py

```diff
@@ -268,16 +269,16 @@ def test_on_before_accelerator_backend_setup(tmpdir):
     configure_optimizers function call."""
 
     class TestCallback(Callback):
-        def on_before_accelerator_backend_setup(self, trainer, pl_module):
-            pl_module.on_before_accelerator_backend_setup_called = True
+        def setup(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", stage: Optional[str] = None) -> None:
```

Comment:
that looks fine to me, just fyi we usually don't add typing to the tests as it adds very little value